client/src/app/pages/dependencies/dependency-apps-table.tsx

```diff
@@ -139,6 +140,15 @@ export const DependencyAppsTable: React.FC<IDependencyAppsTableProps> = ({
     },
   } = tableControls;
 
+  const getManagementValue = (appDependency: AnalysisAppDependency) => {
+    const hasJavaLabel = appDependency.dependency.labels.some((label) => {
+      const labelValue = getParsedLabel(label).labelValue;
+      return labelValue === "java";
+    });
+    const isJavaFile = dependency.name.endsWith(".jar");
```

Comment:
1. Probably better to use `appDependency.dependency.name` just to keep the sourced data in the same place for the function.  Or move the function and the TD render to a column render helper.  Those helpers can be in the same file.  That's what I'm doing in #1540.

2. I don't see any dependency names ending with ".jar".  All the names I see are `mavenGroupId.mavenArtifactId`.  I assume the `appDependency.dependency.provider` string is supposed to be the language, but I've only seen empty string values for my analyses.

Reply:
@sjd78 
@pranavgaikwad mentioned that we only see ".jar" deps when we are using a binary file as the input.

Reply:
Oh that ***is*** an interesting distinction.  

So only "java" things where sources are not available will have a name with a `.jar` suffix.